testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/AcquireTokenFragment.java

```diff
@@ -603,11 +604,17 @@ public void showMessage(String message) {
 
     private void showMessage(final String msg) {
         new Handler(getActivity().getMainLooper()).post(new Runnable() {
-
             @Override
             public void run() {
-                Toast.makeText(getContext(), msg, Toast.LENGTH_LONG).show();
-                mStatus.setText(msg);
+                AlertDialog.Builder builder = new AlertDialog.Builder(getContext());
+                View dialogView = LayoutInflater.from(getContext()).inflate(R.layout.custom_dialog_layout, null);
```

Comment:
why did we create a custom layout? Can we not use default alert dialog? like in below image?

<img width="651" alt="image" src="https://github.com/AzureAD/microsoft-authentication-library-for-android/assets/69237821/124a5760-0676-403e-a6ec-b7e17f6404fc">

Reply:
Also, instead of changing showMessage method, can we add a new showDialog method in which you make this change.. and call showDialog method only in methods which are useful for your testing.

Reply:
ok, could you please post a snapshot of how the dialog looks after your changes?

Reply:
sure, it looks like this:
<img width="409" alt="Screenshot 2023-08-02 at 15 20 21" src="https://github.com/AzureAD/microsoft-authentication-library-for-android/assets/51412053/c6cbb5b0-6b5c-45e1-acf0-387d8f559027">
<img width="410" alt="Screenshot 2023-08-02 at 15 20 07" src="https://github.com/AzureAD/microsoft-authentication-library-for-android/assets/51412053/c67fe2a5-4d62-49d4-afd0-52cf6e9e7f2d">

Reply:
How can we dismiss the dialog? I could not see any ok button in the snpshot

Reply:
we can call `msalTestApp.handleBackButton();` to go back to the main page

Reply:
no, manually how can you dismiss it?

Reply:
We can click anywhere outside the dialog to close the dialog

Reply:
It would be good to have an ok button. Is it a simple thing to add?

Reply:
Added the OK button to the dialog. When the button is clicked the dialog would disappear. 
![Screenshot 2023-08-09 at 14 14 21](https://github.com/AzureAD/microsoft-authentication-library-for-android/assets/51412053/868d888d-fabd-45b1-990f-a0f8c0e1a9d2)